document-service/documents-ui/src/components/documentsTable/index.vue

```diff
@@ -1,15 +1,24 @@
 <script setup lang="ts">
 import { formatToReadableDate } from '~/utils/dateHelper'
-import { documentResultColumns } from '~/utils/documentTypes'
+import { documentTypes, documentResultColumns } from '~/utils/documentTypes'
 import type { DocumentInfoIF } from '~/interfaces/document-types-interface'
 const { getDocumentDescription, downloadFileFromUrl } = useDocuments()
-const { documentList, documentRecord, documentSearchResults, searchEntityId } = storeToRefs(useBcrosDocuments())
+const { documentList, documentRecord, documentSearchResults } = storeToRefs(useBcrosDocuments())
 
 const openDocumentRecord = (searchResult: DocumentInfoIF) => {
   documentRecord.value = { ...searchResult, }
   documentList.value = searchResult.consumerFilenames?.map(file => ({ name: file }))
   navigateTo({ name: RouteNameE.DOCUMENT_RECORDS, params: { identifier: searchResult.consumerDocumentId } })
 }
+
+
+const searchDateRange = ref()
+const searchDocumentClass = ref()
+const documentRecordColumns = ref([])
+
+onMounted(() => {
+  documentRecordColumns.value = documentResultColumns()
+})
```

Comment:
We can probably use documentResultColumns() directly without the hook but we can refine this, as we will tie in with the request for filtered results (or unfiltered). No action required, just a note for future.